src/main/scala/net/facetz/mailru/api/MailRuApiProvider.scala

```diff
@@ -147,4 +151,20 @@ trait MailRuApiProvider extends SimpleLogger {
     }
   }
 
+  protected def deleteRemarketingUsersList(authToken: String, usersListId: Int): Boolean = {
+    val deleteUsersListResponse = Http(s"$clientApiUrl/api/v1/remarketing_users_list/$usersListId.json")
+      .method("DELETE")
+      .addAuthToken(authToken)
+      .setTimeouts()
+      .tryAsString
+
+    if (deleteUsersListResponse.code == 204) {
+      true
+    } else {
+      log.error(s"Code: ${deleteUsersListResponse.code}")
```

Comment:
if you log all errors in tryAsString then you could return just deleteUsersListResponse.code == 204